python/src/test/java/org/apache/zeppelin/python/IPythonInterpreterTest.java

```diff
@@ -88,7 +88,7 @@ public void testGrpcFrameSize() throws InterpreterException, IOException {
     InterpreterContext context = getInterpreterContext();
     result = interpreter.interpret("print(11111111111111111111111111111)", context);
     assertEquals(InterpreterResult.Code.ERROR, result.code());
-    List<InterpreterResultMessage> interpreterResultMessages = context.out.getInterpreterResultMessages();
+    List<InterpreterResultMessage> interpreterResultMessages = context.out.toInterpreterResultMessage();
```

Comment:
maybe rename to    toInterpreterResultMessage**s**

Reply:
Right.

Renaming can be handled in separate issue I think.